actions/pdp-renderer/render.js

```diff
@@ -32,7 +32,7 @@ async function generateProductHtml(sku, urlKey, context) {
   const logger = context.logger;
   let baseProduct;
   if (sku) {
-    const baseProductData = await requestSaaS(ProductQuery, 'ProductQuery', { sku: sku.toUpperCase() }, context);
+    const baseProductData = await requestSaaS(ProductQuery, 'ProductQuery', { sku }, context);
```

Comment:
this will make the `pdp-renderer` action stop working (for generating HTML on individual pages), as it relies on the SKU from the url (always lowercase).
we need a new pattern that works for that as well (e.g. query params)
https://github.com/adobe-rnd/aem-commerce-prerender/blob/main/actions/pdp-renderer/index.js#L64-L66